### DISCUSSION TITLE
Installed but not work.

### DISCUSSION THREAD

**Discussion by aj3423** [nvim-neorg/neorg#409] [General]
I installed neorg, but when opening ".norg" file it still behaves like a normal text file, and command `:Neorg` doesn't exist. 

My configuration using Packer:
```lua
	use {
		"nvim-neorg/neorg",
		config = function()
			require('neorg').setup {
				load = {
					["core.defaults"] = {},
				}
			}
		end,
		requires = "nvim-lua/plenary.nvim"
	}
```
I also tried install "tree-sitter" along with `--after = "nvim-treesitter"` as the "lazy loading" section in readme, still not work.
What do I miss?

**Comment by danymat** [comment]
Hello, can you share your treesitter config (related to Neorg ?) Also, did you do every step in treesitter section in the readme ? (like `:PackerSync`)

**Comment by aj3423** [comment]
@danymat Hi, here's the configuration with treesitter, I did `:TSInstall norg`, `:PackerSync`, `:TSUpdate`.

```lua
	use { 'nvim-treesitter/nvim-treesitter', run = ':TSUpdate' }

	require('nvim-treesitter.configs').setup {
		ensure_installed = { "norg", "markdown" },
		highlight = { -- Be sure to enable highlights if you haven't!
			enable = true,
		}
	}
	use {
		"nvim-neorg/neorg",
		ft = "norg",
		after = "nvim-treesitter",
		config = function()
			require('neorg').setup {
				load = {
					["core.defaults"] = {},
				}
			}
		end,
		requires = "nvim-lua/plenary.nvim"
	}
```
It looks like normal text file.
![image](https://user-images.githubusercontent.com/4710875/160602029-c9f98a7e-f2a3-4774-b497-ea128d3d3476.png)
